src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,107 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/K3rr9q3I2WcXGF3EcDhn/books';
+
+const initialState = {
+  books: [],
+  isLoading: false,
+  error: false,
+  success: false,
+};
+
+const fetchBooks = createAsyncThunk('books/fetchBooks', async () => {
+  try {
+    const res = await axios.get(url);
+    return res.data;
+  } catch (error) {
+    return error;
+  }
+});
+
+const removeBook = createAsyncThunk('books/removeBook', async (id) => {
+  try {
+    const res = await axios.delete(`${url}/${id}`);
+    return res;
+  } catch (error) {
+    return error;
+  }
+});
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231657221-08ffb8c0-0034-4819-9659-60028dedd5da.png)

**[OPTIONAL]**

- [ ] It would be nice if you add commands in the sections shown in the image above to make your readme descriptive enough :+1: